GitUI/CommandsDialogs/FormFileHistory.cs

```diff
@@ -34,6 +34,7 @@ public sealed partial class FormFileHistory : GitModuleForm
         private readonly CancellationTokenSequence _viewChangesSequence = new();
 
         private BuildReportTabPageExtension? _buildReportTabPageExtension;
+        private readonly Dictionary<ObjectId, string> _revisionFilePath = new();
```

Comment:
❔
```suggestion
        private readonly Dictionary<ObjectId, string> _filePathByObjectId = new();
```

Reply:
I prefer the same order as they are declared but using By is maybe used more often, so changing since I should change to objectId anyway